simtools/model/telescope_model.py

```diff
@@ -703,25 +707,25 @@ def isFile2D(self, par):
         bool:
             True if the file is a 2D map type, False otherwise.
         """
-        if not self.hasParameter(par):
+        if not self.has_parameter(par):
             logging.error("Parameter {} does not exist".format(par))
             return False
 
-        fileName = self.getParameterValue(par)
-        file = self.getConfigDirectory().joinpath(fileName)
+        file_name = self.get_parameter_value(par)
+        file = self.get_config_directory().joinpath(file_name)
         with open(file, "r") as f:
             is2D = "@RPOL@" in f.read()
         return is2D
 
-    def readTwoDimWavelengthAngle(self, fileName):
+    def read_two_dim_wavelength_angle(self, file_name):
```

Comment:
we used 2D (or 2d) above and here two_dim. I know they are different functions but let's agree on either one or the other

Reply:
Despite the fact we use 2D, I think this should stay two_dim cause in this case it is spelled out and words should be in lower case.